resources/server/read-test.txt

```diff
@@ -0,0 +1,27 @@
+Here comes the sun (doo doo doo doo)
```

Comment:
Can we differentiate between the read and write test files?

I.e. add "THIS IS THE READ TEST" at the top?